### ISSUE TITLE
tests against dandi-cli failed in master

### ISSUE THREAD

**Issue by yarikoptic** [dandi/dandischema#82]
were fine in #81 .

log shows

```
2021-09-09T20:48:44.6354757Z Creating dandiarchive-docker_django_run ... done
2021-09-09T20:48:46.3178911Z usage: manage.py createsuperuser [-h] [--email EMAIL] [--noinput]
2021-09-09T20:48:46.3180423Z                                  [--database DATABASE]
2021-09-09T20:48:46.3181677Z                                  [--configuration CONFIGURATION] [--version]
2021-09-09T20:48:46.3182819Z                                  [-v {0,1,2,3}] [--settings SETTINGS]
2021-09-09T20:48:46.3183806Z                                  [--pythonpath PYTHONPATH] [--traceback]
2021-09-09T20:48:46.3185479Z                                  [--no-color] [--force-color] [--skip-checks]
2021-09-09T20:48:46.3187073Z manage.py createsuperuser: error: unrecognized arguments: --username admin2
```

may be that is the culprit (didn't check prior logs), but timing is odd ... no updates to containers in the past two days on https://hub.docker.com for any container within `dandiarchive` organization. @jwodder  please investigate

**Comment by jwodder** [comment]
@yarikoptic The dandi-cli integration tests don't use the images on Docker Hub; they build an image with the current version of dandischema as part of the workflow.  If I locally build the dandi-api Docker image from the latest dandi-api code and use that for dandi-cli's tests, the tests fail with the same error shown here.  I suspect that one of dandi-api's dependencies was updated in the past two days in such a way that `createsuperuser` broke.

Comparing the output of `pip list` from the Docker Hub and locally-built versions of the dandi-api image, I find that the following packages have been updated in the past two days:

* anyio
* boto3
* botocore
* dandischema
* django-click
* django-composed-configuration
* django-girder-utils
* sqlparse

@bendichter Are any of the above packages responsible for `createsuperuser --username`?

**Comment by yarikoptic** [comment]
I bet it is coming from django-girder-utils

**Comment by yarikoptic** [comment]
may be this https://github.com/girder/django-composed-configuration/pull/150 is related -- merged 22 hours ago and 0.18.0 was cut with it

**Comment by dchiquito** [comment]
Your solution is correct, the `--username` option was removed by default in Girder 4 apps in favor of always using the email.
